ray-operator/apis/ray/v1/raycluster_types.go

```diff
@@ -161,6 +170,10 @@ const (
 // +kubebuilder:storageversion
 // +kubebuilder:printcolumn:name="desired workers",type=integer,JSONPath=".status.desiredWorkerReplicas",priority=0
 // +kubebuilder:printcolumn:name="available workers",type=integer,JSONPath=".status.availableWorkerReplicas",priority=0
+// +kubebuilder:printcolumn:name="cpus",type=string,JSONPath=".status.desiredCPUs",priority=0
+// +kubebuilder:printcolumn:name="memory",type=string,JSONPath=".status.desiredMemory",priority=0
+// +kubebuilder:printcolumn:name="gpus",type=string,JSONPath=".status.desiredGPUs",priority=0
+// +kubebuilder:printcolumn:name="tpus",type=string,JSONPath=".status.desiredTPUs",priority=1
```

Comment:
> Lower priority (higher numbered) columns will be hidden if the terminal width is too small

it will only show if you use `kubectl get ... -o wide` since I figured it won't be as common and saves screen space.

Reply:
Thanks!